src/bramble/client/StateManager.js

```diff
@@ -0,0 +1,106 @@
+/*global define*/
+/*jslint bitwise: true */
+define(function() {
+    "use strict";    
+
+    var memoryStorage = {
+        _items: {},
+        getItem: function(key) {
+            return memoryStorage._items[key];
+        },
+        setItem: function(key, value) {
+            // Mimic localStorage string storage
+            value = "" + value;           
+            memoryStorage._items[key] = value;
+        },
+        clear: function() {
+            memoryStorage._items = {};
+        }
+    };
+
+    var localStorage = (function(window) {
+        if(typeof window.localStorage === 'undefined') {
+            return memoryStorage;
+        }
+        return window.localStorage;
+    }(window));
+
+    function prefix(name) {
+        return "bramble-property::" + name;
+    }
+
+    function getString(storage, property) {
+        var val = storage.getItem(prefix(property));
+        return val || null;
+    }
+
+    function getBool(storage, property) {
+        var str = getString(storage, property);
+        console.log('getBool', property, str);        
+        if(str === "true") {
+            return true;
+        } else if(str === "false") {
+            return false;
+        } else {
+            // not set
+            return null;
+        }
+    }
+
+    function getInt(storage, property) {
+        var str = getString(storage, property);
+        return str|0;
```

Comment:
It casts the value to a 32-bit Int.  It's like saying `parseInt(str)`, but shorter :)  Doing `str || 0` is not the same thing, even though they both return 0 if `str` is null.

```
> 3.14|0
3
> 3.14||0
3.14
```